alacritty/src/logging.rs

```diff
@@ -108,7 +108,22 @@ impl log::Log for Logger {
         }
 
         let now = time::strftime("%F %T.%f", &time::now()).unwrap();
-        let msg = format!("[{}] [{:<5}] [{}] {}\n", now, record.level(), target, record.args());
+
+        let msg_prefix = format!("[{}] [{:<5}]", now, record.level());
+        let lines = record.args().to_string();
+        let mut lines = lines.split('\n');
+        let msg = format!("{} [{}] {}\n", msg_prefix, target, lines.next().unwrap_or(""));
+
+        let msg = lines.fold(msg, |mut msg, line| {
+            msg.push_str(&format!(
+                "{} {:width$} {}\n",
+                msg_prefix,
+                "",
+                line,
+                width = target.len() + 2,
+            ));
+            msg
+        });
```

Comment:
What's the point of folding here? you got the message already, you're just mutating it anyways? Just use a loop?